### PR TITLE
updating tos errors

### PR DESCRIPTION
**Pull request by BrianCorbin**

### CODE REVIEW

File: TERMS-OF-USE.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ These Terms of Use for MobileCoins and MobileCoin Wallets (Terms) govern:
...
+Please read these Terms carefully before you start to use any MobileCoins and any MobileCoin Wallet. By acquiring or otherwise obtaining MobileCoins, using or transferring MobileCoins or obtaining, accessing or using a MobileCoin Wallet, you acknowledge that you have read, understand, and completely agree to be bound by these Terms. You also agree to require any transferee of your MobileCoins and any holder of a MobileCoin Wallet facilitated by you to be subject to these Terms. These Terms may be enforced against you by MobileCoin TS Ltd., MobileCoin U.S., LLC or other authorized entities (which are collectively referred to in these Terms as the Compliance Entities). These Terms may be amended, changed, or updated by the Compliance Entities at any time and without prior notice to you by posting at [www.buymobilecoin.us]. Your continued use of any MobileCoins and any MobileCoin Wallets following the posting of any amendment, change, or update means that you accept and agree to the amended, changed, or updated Terms. These Terms are first effective as of August 17, 2021 and supersede any previous versions.
```
Comment (jcape) [inline TERMS-OF-USE.md:10]: The `[www.buymobilecoin.us]` text should be `[www.buymobilecoin.us](https://www.buymobilecoin.com/)`

**Review by jcape — CHANGES_REQUESTED** [review]
All the HTTP links should be https, and there are inconsistencies elsewhere in this text about what is used where.

**Review by joekottke — APPROVED** [review]

**Review by sugargoat — APPROVED** [review]

File: TERMS-OF-USE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# **TERMS OF USE**  **FOR MOBILECOINS AND MOBILECOIN WALLETS**
+# **TERMS OF USE** **FOR MOBILECOINS AND MOBILECOIN WALLETS**
```
Comment (the-real-adammork) [inline TERMS-OF-USE.md:1]:
```suggestion
# **TERMS OF USE**  **FOR MOBILECOINS AND MOBILECOIN WALLETS**
```

Comment (the-real-adammork) [reply TERMS-OF-USE.md:1]: this change will make the file equal to the other ones which have 'md5' hash:

`TERMS-OF-USE.md) = 6fc7a4a7a21915add9f36fd20da4e8d3`

**Pull request merged**